mobile-widgets/qmlmanager.cpp

```diff
@@ -155,6 +155,8 @@ void QMLManager::openLocalThenRemote(QString url)
 		if (informational_prefs.unit_system == IMPERIAL)
 			informational_prefs.units = IMPERIAL_units;
 		prefs.units = informational_prefs.units;
+		prefs.tankbar = git_prefs.tankbar;
+		prefs.dcceiling = git_prefs.dcceiling;
```

Comment:
For simplicity yes, I just started with getting setpoint and sensor pO2 visible on Subsurface mobile so it can go inte that part.
The idea is to show setpoint and senosor pO2 instead of the tankbar for CCR dives.